src/Bindings/ManualBindings.cpp

```diff
@@ -3684,6 +3684,48 @@ static int tolua_cEntity_GetSpeed(lua_State * tolua_S)
 
 
 
+static int tolua_cColor_Set(lua_State * tolua_S)
+{
+	cLuaState L(tolua_S);
+
+	if (!L.CheckParamNumber(2, 4) || !L.CheckParamSelf("cColor"))
+	{
+		return 0;
+	}
+	cColor * self = reinterpret_cast<cColor *>(tolua_tousertype(tolua_S, 1, nullptr));
```

Comment:
Doesn't seem to work for me:
```
***curr code for error is  unsigned int
        {
                COLOR_MIN = 0,
                COLOR_MAX = 255,
                COLOR_LIMIT = 256,
                COLOR_NONE = 0xFFFFFFFF,
        };
...
stack traceback:
        [string "../../lib/tolua++/src/bin/lua/basic.lua"]:94: in function 'tolua_error'
        ../../lib/tolua++/src/bin/lua/all.lua:30: in main chunk
        [C]: in function 'dofile'
        BindingsProcessor.lua:55: in function 'invokeToLua'
        BindingsProcessor.lua:1026: in main chunk
        [C]: in ?

** tolua: parse error
stack traceback:
        [C]: in function 'error'
        [string "../../lib/tolua++/src/bin/lua/container.lua"]:787: in function 'doparse'
        [string "../../lib/tolua++/src/bin/lua/container.lua"]:799: in function 'parse'
        [string "../../lib/tolua++/src/bin/lua/class.lua"]:199: in function 'Class'
        [string "../../lib/tolua++/src/bin/lua/container.lua"]:720: in function 'doparse'
        [string "../../lib/tolua++/src/bin/lua/container.lua"]:799: in function 'parse'
        [string "../../lib/tolua++/src/bin/lua/package.lua"]:309: in function 'Package'
        [string "../../lib/tolua++/src/bin/lua/doit.lua"]:55: in function <[string "../../lib/tolua++/src/bin/lua/doit.lua"]:27>
        [C]: in function 'xpcall'
        ../../lib/tolua++/src/bin/lua/all.lua:26: in main chunk
        [C]: in function 'dofile'
        BindingsProcessor.lua:55: in function 'invokeToLua'
        BindingsProcessor.lua:1026: in main chunk
        [C]: in ?.

Code being processed:
unsigned int
```
According to lib/tolua++/include/tolua++.h, it's on version 1.0.92 (I did a submodule update).